projects/plugins/jetpack/extensions/blocks/subscriber-login/edit.js

```diff
@@ -1,10 +1,9 @@
 import { __ } from '@wordpress/i18n';
-import './editor.scss';
 
-function SubscriberLoginEdit() {
+function SubscriberLoginEdit( { className } ) {
 	return (
-		<div>
-			<a href="#">{ __( 'Log out', 'jetpack' ) }</a>
+		<div className={ className }>
+			<a href="#logout-pseudo-link">{ __( 'Log out', 'jetpack' ) }</a>
```

Comment:
Why the `#` wasn't enough?

What do you think about adding `onClick={ e => e.preventDefault() }`?

Reply:
Probably it was enough but the Login/out block is doing it like this and I wanted to be consistent 😄